README.md

```diff
@@ -47,6 +47,22 @@ Below is an example of a bids extension:
         "value": 1,
         "date": "2016-12-10T01:00:00+01:00",
         "notes": "This statistic covers the total number of unique bids received that were disqualified."
+      },
+      {
+        "id": "3",
+        "measure": "highestValidBidValue",
+        "value": 1000,
+        "currency": "USD",
+        "date": "2016-12-12T01:00:00+01:00",
+        "notes": "This statistic covers the total number of unique bids received that were considered valid against relevant criteria."
```

Comment:
I think we can remove the `date` and `notes` fields (`notes` field would need to be updated).

Reply:
I have removed notes and date for `highestValidBidValue` and `lowestValidBidValue` in README